persist_error/handler.py

```diff
@@ -79,18 +80,30 @@ def lambda_handler(event, context):
         }
         warnings.warn(json.dumps(terminal_warning), UserWarning)
 
+        step_function_log_group = f'step-functions-{deploy_stage.lower()}'
+        logs_insights_url = f'https://{region}.console.aws.amazon.com/cloudwatch/home?region={region}#logsV2:logs-insights'
+        logs_insights_query_string = (
+            f'fields @ timestamp, @ message\n'
+            f'| filter @ message like /{execution_arn}/\n'
+            f'| sort @ timestamp desc\n'
+            f'| limit 50'
+        )
```

Comment:
looked up a way to do multi-line f strings here: https://stackoverflow.com/questions/45965007/multiline-f-string-in-python